pom.xml

```diff
@@ -1826,6 +1830,26 @@
       </properties>
     </profile>
 
+    <profile>
+      <!--
+          Use this profile only for making Spark releases. Note that due to SPARK-8819,
+          you must use maven version 3.2.x or before to avoid running into MSHADE-148.
+      -->
+      <id>release-profile</id>
```

Comment:
@pwendell see my comments. `release-profile` is the standard name of the profile triggered by `maven-release-plugin`. I think you're not actually using the release plugin to publish (not an issue right now but did it not work? the script recreates some of what it does) but still might be clearer to use the standard name.